packages/jellyfish-api-core/src/category/masternode.ts

```diff
@@ -0,0 +1,99 @@
+import { ApiClient } from '../.'
+
+export enum MasternodeState {
+  PRE_ENABLED = 'PRE_ENABLED',
+  ENABLED = 'ENABLED',
+  PRE_RESIGNED = 'PRE_RESIGNED',
+  RESIGNED = 'RESIGNED',
+  PRE_BANNED = 'PRE_BANNED',
+  BANNED = 'BANNED'
+}
+
+export class Masternode {
+  private readonly client: ApiClient
+
+  constructor (client: ApiClient) {
+    this.client = client
+  }
+
+  /**
+   * Creates a masternode creation transaction with given owner and operator addresses.
+   *
+   * @param {string} ownerAddress Any valid address for keeping collateral amount
+   * @param {string} operatorAddress  Masternode operator auth address (P2PKH only, unique). If empty, owner address will be used.
+   * @param {CreateMasternodeOptions} options
+   * @param {UTXO[]} [options.inputs = []]
```

Comment:
```suggestion
   * @param {UTXO[]} [options.utxos = []]
```

Reply:
Do you think changing it to `options.utxos` is a good idea? For consistency sake. Take a look at `createMasternodeOptions`

Reply:
Yes we gonna provide a meaningful name, that's actually utxo requested for creating masternode.